server/backend/courses/tests/test_urls.py

```
@@ -0,0 +1,48 @@
+        ("update-delete-course", {"course_id": 1}, UpdateDeleteCourseView),
+        ("update_content", {"id": 1}, UpdateContentView),
+        ("delete_content", {"id": 1}, DeleteContentView),
+        ("enroll_course", {"course_id": 1}, EnrollCourseView),
+        ("get_enrolled_courses", {}, EnrolledCoursesView),
+        ("get_course_content", {"course_id": 1}, CourseContentView),
+        ("instructor-courses", {}, InstructorCoursesView),
+        ("course-rating", {"course_id": 1}, RateCourseView),
+        ("add-content", {}, AddContentAPIView),
+    ]
+)
+def test_course_urls(client, name, kwargs, expected_view):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_course_urls`

Add return type annotation: `None`

(ANN201)